pysatNASA/instruments/cnofs_ivm.py

```diff
@@ -130,12 +130,8 @@ def clean(self):
         max_rpa_flag = 4
         max_dm_flag = 6
 
-    # First pass, keep good RPA fits
-    idx, = np.where(self.data.RPAflag <= max_rpa_flag)
-    self.data = self[idx, :]
-
-    # Second pass, find bad drifts, replace with NaNs
-    idx = (self.data.driftMeterflag > max_dm_flag)
+    idx = ((self.data.driftMeterflag > max_dm_flag)
+           & (self.data.RPAflag > max_rpa_flag))
```

Comment:
```suggestion
    idx = ((self.data.driftMeterflag > max_dm_flag)
           | (self.data.RPAflag > max_rpa_flag))
```
I think I found it @aburrell.  We were excluding points where both data was bad, not either data set.  This explains the "too permissive" effect.  Does this work in your analysis?

Reply:
replaced by later updates